Sources/StreamVideo/Utils/AudioRecorder/AVAudioRecorderBuilder.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import AVFoundation
+import Foundation
+
+actor AVAudioRecorderBuilder {
+
+    // `kAudioFormatLinearPCM` is being used to be able to support multiple
+    // instances of AVAudioRecorders. (useful when using MicrophoneChecker
+    // during a Call).
+    // https://stackoverflow.com/a/8575101
+    static let defaultRecordingSettings: [String: any Sendable] = [
+        AVFormatIDKey: Int(kAudioFormatLinearPCM),
+        AVSampleRateKey: 12000,
+        AVNumberOfChannelsKey: 1,
```

Comment:
Low bitrate and mono audio is currently working as designed?

Reply:
Yeap :)